--- /a/fgsl/configure   2016-08-08 15:50:59.696551740 -0400
+++ /b/fgsl/configure    2016-06-19 03:43:50.000000000 -0400
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 #
 # set up build process for FGSL
 #
@@ -69,7 +69,7 @@
 #
 os=$(uname -s)
 gh=$(which gsl-histogram 2>/dev/null)
-if [ $? != 0 ] ; then
+if [ "$(basename $gh)" != "gsl-histogram" ] ; then
     echo "GSL installation not found. Aborting"
     exit 1
 fi
@@ -236,7 +236,7 @@
        rm -f integer.finc
        ln -s interface/integer_ilp32.finc integer.finc
        ;;
-    int_2_4_[48]_8)
+    int_2_4_8_8)
        rm -f integer.finc
        ln -s interface/integer_ilp64.finc integer.finc
        ;;
