diff --git a/lapack/lapacke/src/Makefile b/lapack/lapacke/src/Makefile
index 36e2869..f1d0213 100644
--- a/lapack/lapacke/src/Makefile
+++ b/lapack/lapacke/src/Makefile
@@ -2053,7 +2053,9 @@ OBJ_FILES := $(C_FILES:.o=.o)
 all: ../../$(LAPACKELIB)
 
 ../../$(LAPACKELIB): $(ALLOBJ) $(ALLXOBJ)
-	$(ARCH) $(ARCHFLAGS) ../../$(LAPACKELIB) $(ALLOBJ) $(ALLXOBJ)
+	# MSYS/MinGW: `ar` can't handle too many arguments, see:
+	# http://trac.haskell.org/gtk2hs/ticket/1146:
+	echo $(ALLOBJ) $(ALLXOBJ) | xargs --max-args=100 $(ARCH) $(ARCHFLAGS) ../../$(LAPACKELIB)
 	$(RANLIB) ../../$(LAPACKELIB)
 
 .c.o:
