diff --git a/xraylib/acc_build b/xraylib/acc_build
index 9eaa5e7..811a3fa 100755
--- a/xraylib/acc_build
+++ b/xraylib/acc_build
@@ -387,7 +387,7 @@ func_install_utilities () {
     if [ ! -e ${UTIL_DIR}/bin/.BUILDCOMPLETE ] ; then
 	echo -e "\nChecking minimium version of GNU Utility Programs needed...\n"
 
-	UTIL_PROG=( "libtool-2.4.2" "m4-1.4.16" "autoconf-2.69" "automake-1.12.2" "makeinfo-4.8" "${CMAKE_UTIL}" )
+	UTIL_PROG=( "libtool-2.4.2" "m4-1.4.16" "autoconf-2.68" "automake-1.12.2" "makeinfo-4.8" "${CMAKE_UTIL}" )
 
 	for UTIL in "${UTIL_PROG[@]}"
 	do
