diff --git a/xraylib/acc_build b/xraylib/acc_build
index 874c4de..564e44e 100755
--- a/xraylib/acc_build
+++ b/xraylib/acc_build
@@ -377,6 +377,10 @@ func_install_utilities () {
         export PATH=/opt/local/libexec/gnubin${PATH:+:${PATH}}
     fi
 
+    if [[ $ACC_COMPILER_TOOLSET == mingw* ]] ; then
+        return
+    fi
+
     export PATH=${UTIL_DIR}/bin:${OTHER_OUTPUT_DIR}/gnu_utilities/bin${PATH:+:${PATH}}
     export LD_LIBRARY_PATH=${UTIL_DIR}/lib:${OTHER_OUTPUT_DIR}/gnu_utilities/lib${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
 
